ui/packages/catalog-ui-search/src/main/webapp/react-component/location/gazetteer.spec.js

```diff
@@ -0,0 +1,20 @@
+import { expect } from 'chai'
+const { getLargestBbox } = require('./gazetteer')
+const test_data = require('./gazetteer-test-data.json')
```

Comment:
❗️ 	It seems excessive to store the entire `gazetteer-test-data.json` file for a single test. Should we hard code the France geojson here instead?